### ISSUE TITLE
Why the results is different?

### ISSUE THREAD

**Issue by houzhong** [rawandahmad698/PyChatGPT#78]
I ask question "What are some well-known, space-based solar telescopes?" via chrome url https://chat.openai.com/chat
It give me result:
```
There are several well-known space-based solar telescopes, including the following:

Solar and Heliospheric Observatory (SOHO): This telescope, launched in 1995, is a collaboration between the European Space Agency (ESA) and NASA. It is designed to study the Sun's structure, composition, and atmosphere, and has made many important discoveries about solar storms, the solar wind, and the Sun's magnetic field.

Solar Dynamics Observatory (SDO): Launched in 2010, the SDO is a NASA mission that observes the Sun's atmosphere and magnetic field in unprecedented detail. It uses a suite of instruments to study the Sun's energy output and the processes that drive space weather near Earth.

Hinode: Also known as Solar-B, Hinode is a Japanese solar observatory launched in 2006. It is designed to study the Sun's magnetic field and its interactions with the solar atmosphere.

Parker Solar Probe: This NASA spacecraft, launched in 2018, is on a mission to study the Sun's atmosphere, known as the corona, up close. It will make several close approaches to the Sun, allowing it to measure the solar wind and other phenomena in unprecedented detail.

Solar Orbiter: This ESA mission, launched in 2020, is designed to study the Sun's magnetic field and its effect on the solar system. It will make close approaches to the Sun_
```
![222](https://user-images.githubusercontent.com/8154412/206722755-5bd50692-9bd4-4fd9-a6d0-5650ed8dba4b.png)

Same question via pychatGPT, the result is very short
`There are many space-based solar telescopes that have been used to study the Sun and its various features. Some well-known examples include the Solar and Heliospheric Observatory (SOHO), the Solar Dynamics Observatory (SDO), the Hinode satellite, and the Interface Region Imaging Spectrograph (IRIS) satellite. These telescopes have provided valuable insights into the Sun's behavior and have helped us better understand how it affects the space environment around Earth.`
![20221209221924](https://user-images.githubusercontent.com/8154412/206722645-feab275d-e14a-4f83-b3de-72b26e19efa9.png)

I have tried many times, get same results

How could get same result like from chrome via pychatgpt?

**Comment by gerrywastaken** [comment]
Similar findings. PyChatGPT:

```shell
> ask "What are some well-known, space-based solar telescopes?"
>> Tracking conversation enabled.
There are several well-known space-based solar telescopes, including the Solar Dynamics Observatory (SDO), the Hinode satellite, and the Solar and Heliospheric Observatory (SOHO). These telescopes are designed to study the sun and its behavior from space, which allows them to observe the sun in ways that are not possible from Earth-based telescopes. The SDO, for example, is capable of capturing high-resolution images of the sun in multiple wavelengths, while the Hinode satellite is equipped with a suite of instruments for studying the sun's magnetic fields and the processes that drive solar activity. SOHO, on the other hand, is a joint project between NASA and the European Space Agency that is designed to study the sun's interior and outer atmosphere.
> ask "What are some well-known, space-based solar telescopes?"
>> Tracking conversation enabled.
There are several well-known space-based solar telescopes that are used by astronomers to study the Sun. Some of the most notable ones include the Solar and Heliospheric Observatory (SOHO), the Hinode satellite, the Solar Dynamics Observatory (SDO), and the Parker Solar Probe. These telescopes are equipped with specialized instruments that allow them to study various aspects of the Sun, such as its magnetic fields, its atmosphere, and its surface features. Because they are located in space, these telescopes are able to obtain high-resolution, unobstructed views of the Sun that are not possible from ground-based telescopes.
> ask "What are some well-known, space-based solar telescopes?"
>> Tracking conversation enabled.
Some well-known space-based solar telescopes include the Solar Dynamics Observatory (SDO), the Solar and Heliospheric Observatory (SOHO), and the Interface Region Imaging Spectrograph (IRIS). These telescopes are designed specifically to study the Sun and its behavior, allowing scientists to learn more about the star at the center of our solar system. They are able to provide detailed images and other data that can help us better understand the processes taking place on and within the Sun.
> ask "What are some well-known, space-based solar telescopes?"
>> Tracking conversation enabled.
Some well-known space-based solar telescopes include the Solar Dynamics Observatory (SDO), the Hinode satellite, and the Solar and Heliospheric Observatory (SOHO). These telescopes are designed specifically to study the Sun and its behavior, and are able to provide detailed images and data about the solar surface and atmosphere. Because they are in space, these telescopes are able to observe the Sun without interference from the Earth's atmosphere, which can distort or block light from the Sun.
```

Firefox:
```
What are some well-known, space-based solar telescopes?

There are several well-known space-based solar telescopes that have been used to study the Sun and its various phenomena. Some examples of these telescopes include:

    The Solar and Heliospheric Observatory (SOHO), which is a joint project of the European Space Agency (ESA) and NASA. SOHO was launched in 1995 and has made many important discoveries about the Sun, including the detection of solar oscillations and the observation of coronal mass ejections.

    The Hinode satellite, which was launched by the Japan Aerospace Exploration Agency (JAXA) in 2006. Hinode is equipped with three telescopes that observe the Sun in different wavelengths, including X-rays and visible light. Hinode has made many important observations of solar activity, including the magnetic field of sunspots and the dynamics of solar flares.

    The Solar Dynamics Observatory (SDO), which is a NASA mission launched in 2010. SDO is equipped with several instruments that observe the Sun in various wavelengths, including ultraviolet and X-ray light. SDO has made many important observations of the Sun, including the discovery of small-scale magnetic fields that contribute to the heating of the solar corona.

    The Parker Solar Probe, which is a NASA mission launched in 2018. The Parker Solar Probe is equipped with several instruments that observe the Sun up close, making measurements of the solar wind and the solar magnetic field. The Parker Solar Probe is the first spacecraft to fly into the Sun's corona, and it has made many important observations of the Sun's atmosphere and solar wind.

These are just a few examples of space-based solar telescopes. There have been many other important missions that have contributed to our understanding of the Sun and its behavior.
```

```csv
The longest answer via PyChatGPT:  752 characters
First answer in Firefox:          1781 characters
```

**Comment by rawandahmad698** [comment]
This is out of scope for the library, as we don't control their endpoint, This library only makes HTTP/S requests

**Comment by rawandahmad698** [comment]
Someone else also reported the same. Nothing I can't fix. I first gotta find a way to do something with a part of the code.

**Comment by gerrywastaken** [comment]
@rawandahmad698 That was fast!

**Comment by transitive-bullshit** [comment]
@rawandahmad698 what specific were you seeing?

I'm not able to discern any noticeable difference in result quality between the web version (using a fresh account for the web version) and the API version. This is using https://github.com/transitive-bullshit/chatgpt-api and it's accompanying [twitter bot](https://github.com/transitive-bullshit/chatgpt-twitter-bot).

The bot is cycling through ~20 different accounts, though, so maybe that's helped to prevent bot detection?

**Comment by acheong08** [comment]
The website has also been lowering the quality of responses:
![image](https://user-images.githubusercontent.com/36258159/206897439-3e1f78d5-95d1-4b06-99f2-97c3726ec825.png)

**Comment by transitive-bullshit** [comment]
I wouldn't be surprised if this is just generally an internal tweak made by OpenAI to help reduce the load on their servers. I haven't seen anything in their discord that says anything about it, though a few people are complaining.

**Comment by rawandahmad698** [comment]
@transitive-bullshit @acheong08 It's out: https://github.com/rawandahmad698/PyChatGPT/blob/master/src/pychatgpt/classes/chat.py#L32

**Comment by rawandahmad698** [comment]
JK, you guys will get the pretty version. DM on Discord or Twitter if you need it

**Comment by rawandahmad698** [comment]
Fixed in 1.0.7
